SygicMapsKit/Classes/Extensions/SYSearchResultExtension.swift

```diff
@@ -28,5 +75,17 @@ extension SYSearchResult: SYUIDetailCellDataSource {
         return UIColor.background
     }
     
-    
+    public func detail(for coordinates: SYGeoCoordinate? = nil, data: @escaping (_ result: SYSearchResultDetail?) -> ()) {
```

Comment:
toto si pyta dokumentacu

Reply:
a mozno by som to dal do samostatnej extension, nie v ramci DetailCellDataSource